### PR TITLE
 adding styling

### PR DESCRIPTION
**Pull request by CodingErik**
adding styling to the page!

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+h1 {
+    color: red;
```
Comment (alimaqsood19) [inline style.css:2]: Looks good, heres a suggestion can we utilize the colour blue because...

File: style.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+}
+
+p {
+    color: blue;
```
Comment (alimaqsood19) [inline style.css:6]:
```suggestion
    color: magenta;
```

File: style.css (hunk 1 above)
Comment (CodingErik) [reply style.css:2]: sure! will look into it ASAP

**Review by alimaqsood19 — APPROVED** [review]
LGTM 👍

**Pull request merged**